.storybook/main.ts

```diff
@@ -28,6 +28,7 @@ module.exports = {
       base,
       define: {
         'process.browser': 'true',
+        'process.env.NODE_DEBUG': undefined,
```

Comment:
Running in a browser, so disable NODE_DEBUG for those scripts that check this flag